lib/ecl/tests/ecl_smspec.c

```diff
@@ -23,6 +23,17 @@
 
 #include <ert/ecl/ecl_smspec.h>
 
+void test_sort( ecl_smspec_type * smspec )
+{
+  ecl_smspec_sort( smspec );
```

Comment:
I quite frankly do not know the number of nodes, but I have added a check verifying that is not changed by the sort.